index.js

```diff
@@ -131,7 +131,8 @@ export default class Carousel extends Component {
         super(props);
         this.state = {
             activeItem: this._getFirstItem(props.firstItem),
-            interpolators: []
+            interpolators: [],
+            oldItemIndex: 0
```

Comment:
@rgabs Thanks for the PR!

In order to avoid triggering `onSnapToItem` on init when `firstItem` is not equal to 0, I think that the initial value of `oldItemIndex` should be the same as `activeItem`, meaning `this._getFirstItem(props.firstItem)`.

Can you please update it?